source/opt/def_use_manager.h

```diff
@@ -63,6 +63,9 @@ class DefUseManager {
   // const overload for ForEachInst().
   void AnalyzeDefUse(ir::Module* module);
```

Comment:
What happens if I call AnalyzeDefUse multiple times?  Does it clear the old data first (I would expect it to.)